### PR TITLE
Adding DIY set guide

### PR DESCRIPTION
**Pull request by Vyolex**
Implementing the setup guide for official DIY set. This is very much still a WIP, publishing it already so people can start providing feedback on what should be included (+ I'm too tired to finish today).

### CODE REVIEW

File: diy_set.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      </td>
+   </tr>
+</table>
+
+## Designing your case
+DIY kits require enclosures to protect them and allow strap attachment. These can be printed, purchased, or crafted.
+
+While the official SlimeVR Main and Extension cases are designed to fit the boards included in this kit, they are not optimised for home 3d printers due to their curved surfaces. As such, we recommend printing up your own using a modified design, or designing your own!
+
+For those planning to design your own enclosure, you can find the model of the Mainboard [here](LINK TO MAINBOARD STEP FILE LOCATION) and Extension PCBs [here](LINK TO EXT STEP FILE LOCATION).
```
Comment (unlogisch04) [inline diy_set.md:49]: List of Cases like in DIY, but that host the Official PCB?

File: diy_set.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+Extensions do not require their own battery, as they use power from the attached Mainboard.
+
+## Choosing your straps
+You have the option of buying, crafting, and/or customising your straps to your personal preferences. We highly recommend you use elasticised straps with hook and loop fastening, to ensure a comfortable experience and prevent tracker migration.
+
+We recommend the following strap lengths as a baseline guide for choosing your strap design:
+* Chest and Hip: 100cm
+* Upper legs (2): 50cm
+* Lower legs (2) and Arms (2): 35cm
+* Feet (2): 30cm
```
Comment (unlogisch04) [inline diy_set.md:68]: Seems to be redundant with the table below?

File: diy_set.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+While the official SlimeVR Main and Extension cases are designed to fit the boards included in this kit, they are not optimised for home 3d printers due to their curved surfaces. As such, we recommend printing up your own using a modified design, or designing your own!
+
+For those planning to design your own enclosure, you can find the model of the Mainboard [here](LINK TO MAINBOARD STEP FILE LOCATION) and Extension PCBs [here](LINK TO EXT STEP FILE LOCATION).
+
+## Choosing your battery
+The SlimeVR Mainboards included in your DIY kit require power from a battery for the best experience. When choosing a battery, we recommend the following specifications:
+* Capacity: 1000-1800 mAh
+* Voltage: 3.7v
+* Connection: Micro JST 1.25mm
+
```
Comment (unlogisch04) [inline diy_set.md:56]: Most important, that they fit into the case.

**Review by unlogisch04 — COMMENTED** [review]
Just some ideas.

File: diy_set.md (hunk 1 above)
Comment (Vyolex) [reply diy_set.md:49]: So far the only case that I am aware off that supports the official PCB is the official case, which will be included as soon as i get the final files for the extensions. I will probably also make an edited version which is more optimized for FDM printers. But we can definitely add this if more cases become available?

File: diy_set.md (hunk 3 above)
Comment (Vyolex) [reply diy_set.md:56]: Yep, was still going to confirm with Eiren which sizes the official case supported, but we will mention that too.

File: diy_set.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+The mainboards feature a Micro JST 1.25mm male connector port for attaching a battery. As such, it is best to choose a battery that has a matching female connector. Alternatively, you can solder or crimp these connectors onto a battery for easy attachment to the Mainboard.
+
```
[the remark this answers is not in the document]
Comment (Vyolex) [reply diy_set.md:58]: Good idea, we'll add an image with polarity indication

File: diy_set.md (hunk 3 above)
Comment (Spazznyan) [reply diy_set.md:56]: Good call. Ill put in "Lithium-ion Polymer (LiPo) pouch batteries come in various shapes and sizes, indicated by an XXYYZZ naming scheme that denotes their dimensions in thickness (X.Xmm), width (YYmm), and length (ZZmm). The battery dimensions will depend on which case you choose to use or make. As an example, the official SlimeVR case design uses an 803443 battery, denoting a battery of 8.0mm thickness, 34mm width, and 43mm length."

**Comment by Vyolex** [comment]
Stuff that still needs to be done:

- [ ] Replace the placeholder images with "proper" ones
- [ ] Add links to official main and extension cases
- [ ] Add link to @TomYumVR 's modified cases (more suited for 3D printing)
- [ ] Add links to pcb 3D models
- [ ] General cleanup

**Comment by calliePepper** [comment]
I am liking this a lot so far. The only thing that I see as a problem is the navigation, the current location isn't in the "DIY Guide" section which causes a bit of confusion, but also it needs to be clearly stated that it is different.

At a quick thought, it should go:

DIY TRACKERS
  - Components Purchase Guide
  - Tracker Schematics
  - IMU Comparison
  - DIY Kit Guide
  - Community-built Case
  - DIY Straps Guide
  
This way it clearly states the components guide is for purchase, and sets it up so that the elements that are important to the kit (case and straps) follow on.

How do you feel that works with it?

**Comment by Vyolex** [comment]
I really like the idea, the only small issue I can see with this is that "normal diy'ers" will then have to skip through a section for DIY kit, unless we can somehow make it skip past. This might be a much more minor issue than I think it would be though. As an additional note, the community-built cases have, as far as i'm aware, no cases that fit the official pcbs.

File: src/diy_set.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+| Lower leg straps   | 2      | 38x350          |
+| Foot straps        | 2      | 25x300          |
+| Arm straps         | 2      | 38x350          |
+</details>
+<br>
+
+## Optional accessories
+SlimeVR DIY Kits Mainboard features a USB Type-C female connector for both recharging and serial connection. As such, at least 1 USB-C cable is required. 
+
+We recommend a USB-C to USB-A cable to charge your SlimeVR trackers, and charging using an external USB-A power supply.
```
Comment (Eirenliel) [inline src/diy_set.md:103]:
```suggestion
We recommend a USB-A to USB-C cable to charge your SlimeVR trackers, and charging using an external USB-A power supply.
```

File: src/diy_set.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+## Choosing your battery
+The SlimeVR Mainboards included in your DIY kit require power from a battery for the best experience. SlimeVR trackers use approximately 100mAh during use. When choosing a battery, **select a capacity rating of at least 500mAh to ensure safe use.** Charging rate is limited to 500mA to ensure optimal battery lifespan. 
+
+We recommend the following specifications:
+* Capacity: 1000-1800 mAh
+* Nominal Voltage: 3.7v
+* Connection: Micro JST 1.25mm
+
+The battery dimensions will depend on which case you choose to use or make. Lithium-ion Polymer (LiPo) pouch batteries come in various shapes and sizes, indicated by an `XXYYZZ` naming scheme that denotes their dimensions in thickness (X.Xmm), width (YYmm), and length (ZZmm). As an example, the official SlimeVR case design uses an 803443 battery, denoting a battery of 8.0mm thickness, 34mm width, and 43mm length.
```
Comment (Eirenliel) [inline src/diy_set.md:59]: Can add that it can fit 804040 or 604040 too.

**Review by Eirenliel — APPROVED** [review]

File: src/diy_set.md (hunk 6 above)
Comment (Spazznyan) [reply src/diy_set.md:59]: yeh I wanted to add the dimensions of the cavity and a few examples so this info is good ty!

**Review by calliePepper — CHANGES_REQUESTED** [review]

**Review by calliePepper — APPROVED** [review]

**Pull request merged**